src/api/solver/content-types/solver/schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "kind": "collectionType",
+  "collectionName": "solvers",
+  "info": {
+    "singularName": "solver",
+    "pluralName": "solvers",
+    "displayName": "Solver"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "name": {
+      "type": "string",
+      "required": true
+    },
```

Comment:
> Can we add solverId field?

No problem

> We could maybe rename this one to displayName

Fine with me, but why not just name? 

> and make it not required.

Why? I would hope they all have name, right?

Reply:
Just `name` is fine.
I thought about being explicit it's not the same name/id used on the backend, which is for display purposes.
And I proposed to make it not required because we can use the solvedId in case one is not provided.